contracts/src/v0.8/keystone/KeystoneForwarder.sol

```diff
@@ -64,15 +69,20 @@ contract KeystoneForwarder is IForwarder, ConfirmedOwner, TypeAndVersionInterfac
 
   bool internal s_reentrancyGuard; // guard against reentrancy
 
+  struct Config {
+    uint8 f; // Number of faulty nodes allowed
+    uint32 minConfigVersion; // oldest config version allowed
+    mapping(uint32 configVersion => OracleSet) oracles;
+  }
+
   /// @notice Contains the signing address of each oracle
   struct OracleSet {
-    uint8 f; // Number of faulty nodes allowed
     address[] signers;
     mapping(address => uint256) _positions; // 1-indexed to detect unset values
   }
 
   /// @notice Contains the configuration for each DON ID
-  mapping(uint32 donId => OracleSet) internal s_configs;
+  mapping(uint32 donId => Config) internal s_configs;
```

Comment:
I expected this to be
```
mapping(bytes32 donConfigId => Config) internal s_configs;
```
where
```
bytes32 donConfigId = keccak256(abi.encode(donId, donConfigVersion))
```

as the `report` function would need a single read to access config (as opposed to 2 reads with the current implementation).

Reply:
Hmm good point, but I still need to store the `minConfigVersion` in a separate mapping then which would also result in a second read. Except now we also spend gas to hash

Reply:
Testing this change locally, it only results in a modest gas savings:

Before:
```
KeystoneForwarder_ReportTest:test_Report_SuccessfulDelivery() (gas: 150908)
```
After:
```
KeystoneForwarder_ReportTest:test_Report_SuccessfulDelivery() (gas: 150820)
```

Reply:
I assume that's because repeated accesses are cheaper after the first one